### ISSUE TITLE
[human_following] wrong install target

### ISSUE THREAD

**Issue by marc-hanheide** [strands-project/strands_hri#80]
see https://lcas.lincoln.ac.uk/jenkins/job/ros-hydro-strands-human-following_binarydeb_precise_amd64/1/console

```
CMake Error at cmake_install.cmake:115 (FILE):
  file INSTALL cannot find
  "/tmp/buildd/ros-hydro-strands-human-following-0.0.4-0precise-20141111-1230/config".
```

so the directory is obviously called `conf` and not `config`. 

In general, please always run `catkin_make install` in your local workspace before pushing any changes.

**Comment by marc-hanheide** [comment]
once this is fixed, please inform @cdondrup for a re-release.

**Comment by cdondrup** [comment]
I'll quickly fix this to also have the meta package installing correctly.

**Comment by ferdianjovan** [comment]
Sorry for that. I did not really try to install it on Bob.

**Comment by cdondrup** [comment]
Closed via #83
